packages/cli/src/commands/styles/lib/radius-styles.ts

```diff
@@ -53,8 +53,17 @@ export const generateGlobalStyles = async ({
   // const retrievedNodes = await getFigmaBlobs(userToken).then((data: FigmaNodeKey[]) => {
   //   return transformNodes(data); 
   // });
-
-  const files = renderTemplate(designTokens);
+  
+  let files: FileTemplate[] = [];
+  // by default now we render all the modules
+  if(template === 'all'){
+    for(const renderer in renderers){
```

Comment:
add typeguard to narrow `renderer` to appropriate type 

https://www.typescriptlang.org/docs/handbook/2/narrowing.html